interfaces/cython/cantera/test/test_convert.py

```diff
@@ -5,19 +5,30 @@
 
 from . import utilities
 import cantera as ct
-from cantera import ck2cti, cti2yaml
+from cantera import ck2cti, ck2yaml, cti2yaml
+
+class converterTestCommon:
```

Comment:
I was going to suggest using `pathlib` here, to move away from string manipulation for paths, but I'm not sure it's worth it. Since the converter functions don't work with path-types, it night as well just still be string stuff...